django_rq/workers.py

```diff
@@ -12,28 +15,55 @@ def get_exception_handlers():
         'EXCEPTION_HANDLERS': ['path.to.handler'],
     }
     """
+    from .settings import EXCEPTION_HANDLERS
+
     return [import_attribute(path) for path in EXCEPTION_HANDLERS]
 
 
+def get_worker_class(worker_class=None):
+    """
+    Return worker class from RQ settings, otherwise return Worker.
+    If `worker_class` is not None, it is used as an override (can be
+    python import path as string).
+    """
+    RQ = getattr(settings, 'RQ', {})
+
+    if worker_class is None:
+        worker_class = Worker
+        if 'WORKER_CLASS' in RQ:
+            worker_class = RQ.get('WORKER_CLASS')
+
+    if isinstance(worker_class, six.string_types):
+        worker_class = import_attribute(worker_class)
+    return worker_class
+
+
 def get_worker(*queue_names, **kwargs):
     """
     Returns a RQ worker for all queues or specified ones.
     """
-    queues = get_queues(*queue_names)
-    name = kwargs.get('name')
-    return Worker(queues,
-                  connection=queues[0].connection,
-                  exception_handlers=get_exception_handlers() or None,
-                  name=name)
+    job_class = get_job_class(kwargs.pop('job_class', None))
+    queue_class = kwargs.pop('queue_class', None)
+    queues = get_queues(*queue_names, **{'job_class': job_class,
```

Comment:
`queues = get_queues(*queue_names, job_class=job_class, queue_class-= queue_class)`

Reply:
That wouldn't work in Python 2.7, see [PEP 3102](https://www.python.org/dev/peps/pep-3102/) which was first implemented in Python 3.